scripts/.mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+
+[mypy-matplotlib.*,networkx.*,pytest.*,numpy.*]
```

Comment:
Based on my understanding, I think we can add more common libraries to .mypy.ini

[mypy-matplotlib.,networkx.,pytest.,numpy.,tensorflow.,torch.,psutil.*]
follow_imports = silent
ignore_missing_imports = true